server/cmd/mmctl/commands/team.go

```diff
@@ -80,10 +80,10 @@ var SearchTeamCmd = &cobra.Command{
 
 // RenameTeamCmd is the command to rename team along with its display name
 var RenameTeamCmd = &cobra.Command{
-	Use:     "rename [team]",
+	Use:     "rename [team] [flag]",
```

Comment:
That is not needed, right?
![Screenshot from 2025-03-26 11-55-59](https://github.com/user-attachments/assets/4548521d-cc66-4a04-a600-73a34d669b80)
_